suite-common/message-system/config/config.v1.json

```diff
@@ -336,14 +290,14 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": "<23.7.2",
+                        "desktop": "<23.9.3",
```

Comment:
Why this change?

Reply:
just a regular bump, see https://www.notion.so/satoshilabs/Secure-message-system-5ade512cbd38481f9d5f65f03dbfd9a8?pvs=4#1d551cfbac9d49218cee628a99426212